plugins/hydra_ray_launcher/example/conf/ray_mode/aws.yaml

```diff
@@ -0,0 +1,24 @@
+# @package _global_
+
+hydra:
+  launcher:
+    sync_up:
+      # source dir is relative in this case, assuming you are running from
+      # <project_root>/hydra/plugins/hydra_ray_launcher/example
+      # absolute path is also supported.
+      source_dir: "."
+      # we leave target_dir to be null
+      # as a result the files will be synced to a temp dir on remote cluster.
+      # the temp dir will be cleaned up after the jobs are done.
+      # recommend to leave target_dir to be null if you are syncing code/artifacts to remote cluster so you don't need
+      # configure $PYTHONPATH on remote cluster
+      include: ["model", "*.py"]
+      # No need to sync up config files.
+      exclude: ["*"]
+    sync_down:
+      include: ["*.pt", "*/"]
```

Comment:
same pattern in both include and exclude?

Reply:
not sure what you meant here, could you clarify?

Reply:
My bad, I linked to the wrong line. Can you check the line below this:

 ```
sync_down:
      include: ["*.pt", "*/"]
      exclude: ["*", "*/"]
```

The `*/` pattern is the same

Reply:
will update :) thx for the patience :P